Job Positions (vacatures)

StefanKroon opened the pull request.

vijivasanthan committed:
New corner ribbon added for vacatures and page for job description links created.

vijivasanthan committed:
teamup link removed

vijivasanthan committed:
forward '/' removed in link in index page

vijivasanthan committed:
removed spaces

vijivasanthan committed:
changed Url

vijivasanthan committed:
changed Url

Commit:
no change

Commit:
no change

vijivasanthan committed:
changed vacature text color

Commit:
no change

Commit:
no change

vijivasanthan committed:
changed size of the header box

Commit:
no change

Commit:
no change

vijivasanthan committed:
Styling changes and code alignment changed

StefanKroon reviewed (commented):
Nothing found. Done

Commit:
styling done
Merge branch 'develop' of https://github.com/askcs/team-up-promo into develop

StefanKroon commented on Vacatures.html:
```diff
@@ -0,0 +1,301 @@
+    <meta name="description" content="">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+
+    <meta property="og:title" content="TeamUp" />
```
Should be TeamTelefoon

StefanKroon commented on Vacatures.html:
```diff
@@ -0,0 +1,301 @@
+
+    <meta property="og:title" content="TeamUp" />
+    <meta property="og:type" content="image" />
+    <meta property="og:url" content="http://teamup.ask-cs.nl/" />
```
Should link to TeamTelefoon. @vijivasanthan Where did you copy this template from? Not from indexTeamTelefoon.html?

vijivasanthan replied:
I copied it from indexTeamUp.html

StefanKroon commented on Vacatures.html:
```diff
@@ -0,0 +1,301 @@
+<!-- end navbar -->
+
+
+<div class="intro" >
```
There is too much "empty space" above the "Vacatures" title.

StefanKroon commented on Vacatures.html:
```diff
@@ -0,0 +1,301 @@
+            </div>
+        </div>
+    </div>
+<br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br>
```
Are all these <br> really necessary?

StefanKroon commented on Vacatures.html:
```diff
@@ -0,0 +1,301 @@
+    <p>
+
+        Heemraadssingel 89 | 3022 CA Rotterdam | Nederland | 085 888 4445 |
+        <a style="font-size: 16px" href="mailto:contact@teamtelefoon.nl">contact@teamtelefoon.nl</a></p>
```
Why is there a seperate font-size tag necessary here? Can't do without?

StefanKroon commented on promo/js/main.js:
```diff
@@ -15,6 +15,48 @@ jQuery(function() {
+    vacatureItem = '';
+
+    if (jobclassName == 'job2' && vacatureItem == undefined || vacatureItem == '') {
+      jQuery('#TextContainer1').hide();
```
I think "TextContainer1" is a bad name. Also "TextContainer2" is not very desriptive

StefanKroon commented on promo/js/main.js:
```diff
@@ -15,6 +15,48 @@ jQuery(function() {
+  });
+
+
+  jQuery('.vacatures-item').click(function () {
```
This function should probably close the job position on a second click. The code is never executed, I think.

StefanKroon reviewed (commented):
@vijivasanthan Can you take a look to my 7 comments?

vijivasanthan committed:
fixed styling in few places and enhanced the view.

Commit:
styling changes done
Merge branch 'develop' of https://github.com/askcs/team-up-promo into develop

Conflicts:
	sass/main.scss

Commit:
Added fabric-pattern image which are part of the Voicemail-emails to become undependent of ASK-Fast

Commit:
Made some stylesheet changes, but now javascript is not working well anymore.

vijivasanthan committed:
includes css

vijivasanthan committed:
changes in script for making links work well.

vijivasanthan committed:
changes in script for making links work well.

Commit:
Updated the job position texts
